src/site/content/en/learn/css/transitions/index.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Transitions
+description: >
+  In this module, learn how to define transitions between states of an element. Use transitions to improve user experience by providing visual feedback to user interaction.
+  
+audio:
+  title: 'The CSS Podcast - 044: Transitions'
+  src: https://traffic.libsyn.com/secure/thecsspodcast/TCP053_v1.mp3?dest-id=1891556
+  thumbnail: image/foR0vJZKULb5AGJExlazy1xYDgI2/ECDb0qa4TB7yUsHwBic8.png
+authors:
+  - cambickel
+date: 2021-09-15
+---
+
+When interacting with a website, you might notice that many elements have _state_. For example, dropdowns can be in opened or closed states. Buttons might change color when focused or hovered. Modals appear and disappear.
+
+By default, CSS switches the style of these states instantly.
+
+Using CSS transitions, we can interpolate between the initial state and the target state of the element. The transition between the two enhances the user experience by providing visual direction, support, and hints about the cause and effect of the interaction.
+
+{% Aside 'key-term' %}
+_Interpolation_ is the process of creating "in-between" steps that smoothly transition from one state to another.
+{% endAside %}
+
+{% Codepen {
+  user: 'web-dot-dev',
```

Comment:
Recommendation: Have some action text for this demo so users know they're supposed to hover over the element

Reply:
Done, good idea!